courses/CSF211/README.md

```diff
@@ -11,7 +11,6 @@ DSA is really the bread and butter for good programming practices. Organizing co
 *   [Code](#code)
 *   [Videos](#videos)
 *   [Websites](#websites)
-*   [Communities](#communities)
```

Comment:
`Communities` link wouldn't have to be removed, but rather a new subheading `Communities` with a list of quality DSA communities could be added